com.unity.render-pipelines.high-definition/Editor/Wizard/HDWizard.Configuration.cs

```diff
@@ -144,57 +144,106 @@ public Entry(QualityScope scope, InclusiveMode mode, Style.ConfigStyle configSty
             }
         }
 
+
+        [InitializeOnLoadMethod]
+        static void Bobby()
+        {
+            if (EditorWindow.HasOpenInstances<HDWizard>())
+            {
+                HDWizard window = (HDWizard)EditorWindow.GetWindow(typeof(HDWizard));
+                window.ReBuildEntryList();
+            }
+        }
+
+        Entry[] BuildEntryList()
```

Comment:
[Optional] I think that you can simplify things a little by removing BuildEntryList() and only keep ReBuildEntryList().

You should be able to make it work with only 
```
internal void ReBuildEntryList()
{
    //compute List<Entry> entryList as below this comment
    m_Entries = BuildEntryList();
}
```

and then entries would become
```
Entry[] entries
{
    get
    {
        // due to functor, cannot static link directly in an array and need lazy init
        if (m_Entries == null)
            ReBuildEntryList();
        return m_Entries;
    }
 }
```
Especially because when cleaning current ReBuildEntryList, it will become a one liner method.